docker/jenkins.yml

```diff
@@ -39,7 +39,7 @@ jenkins:
   - "Ping"
   authorizationStrategy:
     github:
-      adminUserNames: "MancunianSam,Dclipsham,ian-hoyle,LauraDamianTNA,MarkKingsbury,plembke-tna,sarrahdi,suzannehamilton,TomJKing"
+      adminUserNames: "MancunianSam,Dclipsham,ian-hoyle,LauraDamianTNA,MarkKingsbury,plembke-tna,sarrahdi,suzannehamilton,TomJKing,paulschwarzenberger"
```

Comment:
Did you ever look into whether we could use a GitHub group to manage these permissions, so we don't have to configure everyone username? It would make it easier to manage leavers, too, since they'll be removed from the GitHub org anyway.

Reply:
No, I don't think you can, there's nothing in the plugin documentation to say we can. The leavers thing isn't such a problem because we're restricting access to Jenkins to only the office for now so anyone leaving won't get access.